README.md

```diff
@@ -1,79 +1,285 @@
+[![GitHub Issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-library/module/azure-servicebus.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-library/labels/module/azure-servicebus)
 
-The [Azure Service Bus](https://azure.microsoft.com/en-us/services/service-bus/) is a reliable cloud Messaging as a
-Service (Maas). It is an enterprise message broker with message queues and publish-subscribe topics. The service bus is
-used to decouple applications and services from each other. Data is transferred between different applications and
-services using messages. You can find more
-information [here](https://docs.microsoft.com/en-us/azure/service-bus-messaging/).
+## Overview
 
-The Azure Service Bus [Ballerina](https://ballerina.io/) connector allows you to connect to the Azure Service Bus to
-send and receive messages. You can perform actions such as send to queue, send to topic, receive from queue, receive
-from subscription, etc.
+The [Ballerina](https://ballerina.io/) connector for Azure Service Bus allows you to connect to
+an [Azure Service Bus](https://docs.microsoft.com/en-us/azure/service-bus-messaging/) via the Ballerina language.
 
-For more information, go to the module(s).
+The Azure Service Bus is a fully managed enterprise message broker with message queues and publish-subscribe topics. It
+provides the capability to send and receive messages from Service Bus queues, topics, and subscriptions. The Azure
+Service Bus handles messages that include data representing any kind of information, including structured data encoded
+with common formats such as the following ones: JSON, XML, and Plain Text.
 
-- [asb](ballerina/Module.md)
+This connector supports the following operations:
+- Manage (Get/Create/Update/Delete/list) a queue, topic, subscription or rule.
+- Send messages to a queue, topic, or subscription.
+- Receive messages from a queue, topic, or subscription.
 
-## Building from the source
+Service Bus provides a Microsoft-supported [native Java API](https://docs.microsoft.com/en-us/java/api/overview/azure/servicebus?view=azure-java-stable) (
+SDK) and this module makes use of
+this [public API](https://learn.microsoft.com/en-us/java/api/com.microsoft.azure.servicebus?view=azure-java-archive)
+. As the public API
+applies [SAS authentication](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-sas), this module
+supports SAS authentication as well.
 
-### Setting up the prerequisites
+This module
+supports [Service Bus SDK 7.13.1 version](https://learn.microsoft.com/en-us/java/api/overview/azure/service-bus?view=azure-java-stable#libraries-for-data-access)
+. The source code on GitHub is
+located [here](https://github.com/Azure/azure-service-bus-java). The
+primary wire protocol for Service Bus is Advanced Messaging Queueing Protocol (AMQP) 1.0, an open ISO/IEC standard.
```

Comment:
I think the supported SDK version should be included in a table for better readability.

Reply:
Changed to more simplified content with @NipunaRanasinghe's suggestions [ff89865](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/206/commits/ff89865bcc09cd1e41a02929615f7d6cff1ffdca).